packages/cysync-core-constants/src/i18n/lang/en.json

```diff
@@ -1659,7 +1659,7 @@
           "subTitle": "Authenticating the wallet ",
           "actions": {
             "confirmAuth": "Confirm wallet authentication on device",
-            "enterPinAndTapCard": "Enter Pin and tap any one card"
+            "enterPinAndTapCard": "Tap any one card"
```

Comment:
Isn't this also required for estate recovery?

Reply:
No estate recovery is strings are as per figma design